xql/src/xql/utils.py

```diff
@@ -26,3 +30,25 @@ def wrap(*args, **kw):
         print(f"Query took: { strftime('%H:%M:%S', gmtime(te - ts)) }")
         return result
     return wrap
+
+
+def connect_dask_cluster() -> None:
+    """
+    Connects to a Dask cluster.
+    """
+    # Fetch the cluster name from environment variable, default to "era5-dask" if not set
+    cluster_name = os.getenv('DASK_CLUSTER', "xql-dask")
```

Comment:
nit: let's mention about this env variable in readme.

Reply:
Previous Comment.